CHANGELOG.md

```diff
@@ -40,18 +40,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## v16.0.0
```

Comment:
Could you please also review our v16 upgrade handler and make sure that we specify all parameter updates in v16 here?

Reply:
Please ping me on Slack once complete

Reply:
updated!